tools/compatgen/internal/tensorflow.go

```diff
@@ -12,6 +13,7 @@ import (
 
 func FetchTensorFlowCompatibilityMatrix() ([]config.TFCompatibility, error) {
 	url := "https://www.tensorflow.org/install/source"
+	min_cuda_version := "10.1"
```

Comment:
Should this be defined as a const at the package level outside of the functions? 

How do you determine the min CUDA version and is that something that will need to be updated on a semi-frequent basis?

Reply:
How about making minimum CUDA version a parameter of this method, and then defining a constant at the call site?

Reply:
yeah, those are good points. I will add the min CUDA version to the compatibility package.

Reply:
Shall we say 11.0 is the minimum? It was released in 2020, and I doubt people will build new images with CUDA<11